index.d.ts

```diff
@@ -246,7 +246,7 @@ interface FabProps extends TouchableProps {
 
 interface FlashListProps extends RNFlashListProps {
   SkeletonComponent?: React.FC;
-  durationPerItem?: number;
+  animationDuration?: number;
```

Comment:
@bigbinary/neeto-rn Now onwards whenever we add or update props we need to also update type file (index.d.ts)